.github/workflows/per-pr.yml

```diff
@@ -32,6 +32,7 @@ jobs:
             "./.github/tests/mev.yaml",
             "./.github/tests/mev-mock.yaml",
             "./.github/tests/mix-with-tools.yaml",
+            "./.github/tests/mix-persistence.yaml",
```

Comment:
this fails for docker due to volume mount issues; rwx-r-r but need 777

Reply:
I'd argue that this is a blocker for this PR.